single_tech_samples/streamanalytics/template-parameters-generator.py

```diff
@@ -0,0 +1,123 @@
+import json
```

Comment:
I think a template generator is out of scope. We can get away with just having a yaml with pipeline variables.